### DISCUSSION TITLE
A default route is present but there is no public prefix on lan thus we don't announce a default route by setting ra_lifetime to zero!

### DISCUSSION THREAD

**Discussion by maxy1** [immortalwrt/immortalwrt#1290] [Q&A]
### Describe the bug

My System logs full with above message. 
![Capture](https://github.com/immortalwrt/immortalwrt/assets/22884474/8bd7c3c9-49ae-4f2f-b6da-41a74ce93734)

Here is the result of # ip a
`root@ImmortalWrt:~# ip a
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group defaul                                           t qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host
       valid_lft forever preferred_lft forever
2: sit0@NONE: <NOARP> mtu 1480 qdisc noop state DOWN group default qlen 1000
    link/sit 0.0.0.0 brd 0.0.0.0
3: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq master br-lan state                                            UP group default qlen 1000
    link/ether d4:20:00:b2:47:bf brd ff:ff:ff:ff:ff:ff
4: eth1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP group defa                                           ult qlen 1000
    link/ether d4:20:00:b2:47:c0 brd ff:ff:ff:ff:ff:ff
    inet 192.168.1.165/24 brd 192.168.1.255 scope global eth1
       valid_lft forever preferred_lft forever
    inet6 2402:4000:2340:6b0b:d620:ff:feb2:47c0/64 scope global dynamic noprefix                                           route
       valid_lft 86177sec preferred_lft 14177sec
    inet6 fe80::d620:ff:feb2:47c0/64 scope link
       valid_lft forever preferred_lft forever
5: br-lan: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP gro                                           up default qlen 1000
    link/ether d4:20:00:b2:47:bf brd ff:ff:ff:ff:ff:ff
    inet 192.168.5.1/24 brd 192.168.5.255 scope global br-lan
       valid_lft forever preferred_lft forever
    inet6 fde8:c5e3:e1db::1/60 scope global noprefixroute
       valid_lft forever preferred_lft forever
    inet6 fe80::d620:ff:feb2:47bf/64 scope link
       valid_lft forever preferred_lft forever
`
Here is the result of # ip -6 route

`root@ImmortalWrt:~# ip -6 route
default from 2402:4000:2340:6b0b::/64 via fe80::d472:26bc:f3a7:1 dev eth1 proto static metric 640 pref medium
2402:4000:2340:6b0b::/64 dev eth1 proto static metric 256 pref medium
unreachable 2402:4000:2340:6b0b::/64 dev lo proto static metric 2147483647 pref medium
fde8:c5e3:e1db::/64 dev br-lan proto static metric 1024 pref medium
unreachable fde8:c5e3:e1db::/48 dev lo proto static metric 2147483647 pref medium
fe80::/64 dev br-lan proto kernel metric 256 pref medium
fe80::/64 dev eth1 proto kernel metric 256 pref medium
`

### ImmortalWrt version

r27625-416c8c5c91

### ImmortalWrt release

23.05.2

### ImmortalWrt target/subtarget

x86/64

### Device

To be filled by O.E.M. To be filled by O.E.M.

### Image kind

Official downloaded image

### Steps to reproduce

This message is coming up in logs 

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by KardRi** [comment]
Wan doesn't delegate ipv6 prefix, thus there is no ipv6 in LAN.
